lib/ansible/modules/system/aix_filesystem.py

```diff
@@ -0,0 +1,577 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author:
+  - Kairo Araujo (@kairoaraujo)
+module: aix_filesystem
+short_description: Configure LVM and NFS file systems for AIX
+description:
+  - This module creates, removes, mount and unmount LVM and NFS file system for
+    AIX using C(/etc/filesystems).
+  - For LVM file systems is possible to resize a file system.
+version_added: "2.5"
+options:
+  account_subsystem:
+    description:
+      - Specifies whether the file system is to be processed by the accounting
+        subsystem.
+    type: bool
+    default: "no"
+  attributes:
+    description:
+      - Specifies attributes for files system separated by comma.
+    default: agblksize='4096',isnapshot='no'
```

Comment:
Since recently we expect module parameters to have the type declared in the documentation.
So this one would state:
```yaml
    type: str
```